pandas/core/groupby.py

```diff
@@ -1475,9 +1475,88 @@ def describe(self, **kwargs):
     @Substitution(name='groupby')
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
-        """
-        Provide resampling when using a TimeGrouper
+        """Provide resampling when using a TimeGrouper.
```

Comment:
The short summary goes to the next line. Did you run the `./validate_docstrings.py` script with it? It should let you detect formatting errors like this, and save time from reviewers, on things that a script can tell you.